make/compiler/Makefile.cray-prgenv

```diff
@@ -46,10 +46,19 @@ GEN_DYNAMIC_FLAG = -dynamic
 LIB_STATIC_FLAG =
 LIB_DYNAMIC_FLAG = -shared
 
-# With locale models other than 'flat' we will need libnuma.  That is
-# so far available only in .so form, but the PrgEnv compilers default
-# to forcing static linking.  So, override them in this case.
+#
+# Circumstances may push us to choose dynamic linking instead of the
+# PrgEnv default static linking.
```

Comment:
The default link method is site-specific, so maybe change the wording a little bit?

Reply:
Ah, I'd forgotten that detail and the original comment didn't mention it to remind me!  I'll change the new comment.